### PR TITLE
Created Class Camel and TestCamel and made changes to Trader.

### PR DESCRIPTION
**Pull request by Winnie1029**

### CODE REVIEW

File: src/main/java/Trader.java [hunk 1]
```diff
@@ -96,7 +96,9 @@ public static void main(String[] args) {
         List<Tradable> all_items = Arrays.asList(
                 new Horse(),
                 new Horse(),
-                new Horse()
```
Comment (Jennylin331431) [inline src/main/java/Trader.java]: missing commma

File: src/main/java/TestCamel.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    @Test(timeout = 50)
+    public void TestSound() {
+        assertEquals("Nuzz!", c.sound());
+    }
+
+    @Test(timeout = 50)
+    public void TestGetMaxSpeed() {
+        assertEquals(1, c.getMaxSpeed());
```
Comment (paulslss300) [inline src/main/java/TestCamel.java:26]: assertEquals(2, c.getMaxSpeed())?

**Review by Jennylin331431 — CHANGES_REQUESTED** [review]

File: src/main/java/TestCamel.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    }
+
+    @Test(timeout = 50)
+    public void TestUpgradeSpeed() {
+        c.upgradeSpeed();
+        assertEquals(2, c.getMaxSpeed());
```
Comment (paulslss300) [inline src/main/java/TestCamel.java:32]: can you confirm your test cases pass?

**Review by DavidM0808 — APPROVED** [review]
Everything looks good here.

**Review by paulslss300 — CHANGES_REQUESTED** [review]
see my comments.

**Pull request merged**